README.md

```diff
@@ -1,2 +1,36 @@
-# elrond-actions
-Github actions for smart contracts
+# Github Actions for smart contracts
+
+A Github Action for smart contracts which:
+- builds the wasm files
+- runs mandos-rs and mandos-go tests
+- does a clippy check
+- provides a report containing details about the smart contracts
+
+## Usage
+
+### Standard build
+
+See [contracts.yml](.github/workflows/contracts.yml)
+This uses fixed versions of rust and vmtools.
+Ignores `eei` checks which allows the contracts to use features which are not live on the elrond mainnet yet.
+
+```yml
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  contracts:
+    name: Contracts
+    uses: ElrondNetwork/elrond-actions/.github/workflows/contracts.yml@v1
+    with:
```

Comment:
At the moment the aim is not to make the build deterministic (anyway, this is done through docker images), but only to port the existing functionality and to integrate `erdpy contract report`. The latest `wasm-opt` should be fine.
The versions which are fixed at the moment (rust nightly, vmtools) are fixed because we've had broken builds in the past because of changes in those two.
The rust nightly is volatile and since our code depends on a lot of features. For vmtools the outputs of the mandos tests may differ - so upgrades have to be done manually.
I added the possibility to specify the `erdpy` version and added information about the extra option to the `README.md`